manual-workflows/resources.sh

```diff
@@ -102,7 +126,7 @@ sh $SRC_DIR/../scripts/enable_api.sh
 case $COMMAND in
     "init-project")
         # Create service accounts
-        sh $SRC_DIR/../scripts/create_resources.sh $PROJECT $SERVER_NAME $COMPUTE_NAME $BUCKET
+        sh $SRC_DIR/../scripts/create_resources.sh $PROJECT $SERVER_NAME $COMPUTE_NAME $BUCKET $CIDR $GC_REGION
         # Create bucket if not exists
         # Generate cromwell.conf
         generate_config
```

Comment:
It seems like it might be nice to have this config somewhere other than the directory that contains the scripts. If I run this docker image on the cluster, then the source directory in /opt/ is non-writeable, which is a pain in the butt.  (similarly, I think the `start.sh` script would then need to take config path as an option properly).  Perhaps this belongs in a separate issue, though?